man/bayesplot-package.Rd

```diff
@@ -4,60 +4,57 @@
 \name{bayesplot-package}
 \alias{bayesplot-package}
 \alias{bayesplot}
-\title{Plots for Bayesian Models}
+\title{\strong{bayesplot}: Plotting for Bayesian Models}
 \description{
 \if{html}{
    \figure{stanlogo.png}{options: width="50px" alt="mc-stan.org"}
-   \emph{Stan Development Team}
 }
+\emph{Stan Development Team}
 
-The \pkg{bayesplot} package provides a variety of \pkg{ggplot2}-based
+The \strong{bayesplot} package provides a variety of \strong{ggplot2}-based
 plotting functions for use after fitting Bayesian models (typically, though
 not exclusively, via Markov chain Monte Carlo). The package is designed not
 only to provide convenient functionality for users, but also a common set of
 functions that can be easily used by developers working on a variety of
 packages for Bayesian modeling, particularly (but not necessarily) packages
-powered by \pkg{\link[rstan]{rstan}}. Examples of packages that will soon (or
-already are) using \pkg{bayesplot} are \pkg{rstan} itself, as well as the
-\pkg{rstan}-dependent \pkg{rstanarm} and \pkg{brms} packages for applied
-regression modeling.
+powered by \link[rstan:rstan-package]{rstan} (the \R interface to Stan).
+Examples of packages that will soon (or already are) using \strong{bayesplot} are
+\strong{rstan} itself, as well as the \strong{rstan}-dependent \strong{rstanarm} and
+\strong{brms} packages for applied regression modeling.
 }
```

Comment:
This is just an observation... I'm noticing that all the \pkg tags turn into \strong codes. I guess that will cause them to turn bold on the pkgdown site too, which is fine.

Reply:
That's actually something that I did manually. roxygen2md leaves them as `\pkg{pkgname}` in the roxygen doc, but having `\pkg{pkgname}` in an Rd file has the effect of making the package name bolded when viewing the doc anyway so I just switched it to `**pkgname**`.